ext/http/response_body.rs

```diff
@@ -30,10 +29,6 @@ pub enum ResponseStreamResult {
   /// not register a waker and should be called again at the lowest level of this code. Generally this
   /// will only be returned from compression streams that require additional buffering.
   NoData,
-  /// Stream provided trailers.
-  // TODO(mmastrac): We are threading trailers through the response system to eventually support Grpc.
-  #[allow(unused)]
-  Trailers(HeaderMap),
```

Comment:
Why remove this one?

Reply:
It was unused -- we ended up doing trailers a different way 🙃